src/modules/components/dao/daoAvatar/daoAvatar.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
+import type React from 'react';
+import { Avatar, type IAvatarProps } from '../../../../core';
+
+export interface IDaoAvatarProps extends Omit<IAvatarProps, 'fallback'> {
+    /**
+     * Name of the DAO
+     */
+    daoName: string;
```

Comment:
Given that we're already in the dao context, I'd suggest to simply name the property as `name`. Similar for the DaoListItem properties (name, description, logo, ..)
```suggestion
    name: string;
```